src/devhub/devhub.js

```diff
@@ -51,11 +51,14 @@ async function mainMetrics() {
   const dataUrl =
     "https://raw.githubusercontent.com/tigerbeetle/devhubdb/main/devhub/data.json";
   const data = await (await fetch(dataUrl)).text();
+  const maxBatches = 200;
   const batches = data.split("\n")
     .filter((it) => it.length > 0)
-    .map((it) => JSON.parse(it));
+    .map((it) => JSON.parse(it))
+    .slice(-1 * maxBatches);
```

Comment:
Nice! This’ll also naturally evict old timelines we no longer update!

the long term plan is also to log-rotate the actual JSON file manually once a year or so